### DISCUSSION TITLE
RNN, LSTM, GRU

### DISCUSSION THREAD

**Discussion by Yingxiu-Chang** [bitcraze/discussions#896] [General]
Hi all, 

I'm trying to run the custom netwotks on Gap8. 
I confuse about is it able to run the GRU netwotks on GAP8?

Best regards
Hugh

**Comment by krichardsson** [comment]
The best source for what the GAP8 can do is to check out GreenWaves' documentation.
https://greenwaves-technologies.com/gap8_mcu_ai/
https://github.com/GreenWaves-Technologies/gap_sdk
